confd/templates/opennms-activemq.xml.tpl

```diff
@@ -0,0 +1,196 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<!-- DO NOT CHANGE THIS FILE! GENERATED FROM confd -->
+
+<beans
+  xmlns="http://www.springframework.org/schema/beans"
+  xmlns:amq="http://activemq.apache.org/schema/core"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+  http://activemq.apache.org/schema/core http://activemq.apache.org/schema/core/activemq-core.xsd">
+
+    <!-- Allows us to use system properties as variables in this configuration file -->
+    <bean class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
+        <property name="systemPropertiesModeName" value="SYSTEM_PROPERTIES_MODE_OVERRIDE" />
+        <!-- By default, use the data/tmp/activemq directory for on-disk storage -->
+        <property name="properties">
+             <value>
+                 activemq.data=${opennms.home}/data/tmp/activemq
+             </value>
+        </property>
+    </bean>
+
+    <!--
+        The <broker> element is used to configure the ActiveMQ broker.
+    -->
+    <broker xmlns="http://activemq.apache.org/schema/core" brokerName="localhost" dataDirectory="${activemq.data}" deleteAllMessagesOnStartup="{{getv "/activemq/broker/delete-all-messages-on-startup" "false"}}">
+        <plugins>
+          <bean id="openNMSJaasBrokerPlugin" class="org.opennms.netmgt.activemq.auth.OpenNMSJaasBrokerPlugin" xmlns="http://www.springframework.org/schema/beans" />
+
+          <authorizationPlugin>
+            <map>
+              <authorizationMap>
+                <authorizationEntries>
+                  {{$auth := "/activemq/authorization-entry" -}}
+                  <!-- Users in the admin role can read/write/create any queue/topic -->
+                  <authorizationEntry queue=">" read="{{getv (print $auth "/queue/read") "admin"}}" write="{{getv (print $auth "/queue/write") "admin"}}" admin="{{getv (print $auth "/queue/admin") "admin"}}" />
```

Comment:
is `queue=">"` the correct value here? Just double checking since that looks a bit strange but I'm not familiar with this file. Same on line `52`.

Reply:
Yes, that's correct. This file was not created, it was just enhanced.
[This](https://github.com/OpenNMS/opennms/blob/develop/opennms-base-assembly/src/main/filtered/etc/opennms-activemq.xml#L47) is the reference content.